Library/Formula/ttygif.rb

```diff
@@ -0,0 +1,17 @@
+class Ttygif < Formula
+
+  desc "Convert terminal recordings to animated gifs"
+  homepage "https://github.com/icholy/ttygif.git"
+  url "https://github.com/icholy/ttygif/archive/1.0.8.tar.gz"
+  sha256 "32b3394ebaac3389c66aee225ab61846fc84b02e218d0018515a6e9345a9f114"
+  head "https://github.com/icholy/ttygif.git", :branch => "master"
```

Comment:
Don't need to specify `:branch => "master"`. We assume that to be the default unless told otherwise.